.gitignore

```diff
@@ -1,3 +1,4 @@
 /composer.lock
 /vendor
 /bin
+.idea
```

Comment:
Could you remove this line from this PR? I think that it belongs more in a global, personal .gitignore :)